src/tsdf/write_binary.py

```diff
@@ -19,12 +19,10 @@ def write_dataframe_to_binaries(
     """
     Save binary file based on the provided pandas DataFrame.
 
-    :param file_dir: path to the directory where the file will be saved.
-    :param file_name: name of the file to be saved.
-    :param data: pandas DataFrame containing the data.
-    :param metadata: dictionary containing the metadata.
-
-    :return: TSDFMetadata object.
+    :param file_dir:    path to the directory where the file will be saved.
```

Comment:
We could at one point convert all the documentation to NumPy style (we started it for the `dbpd`), just not to forget.